lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -24,6 +24,10 @@ class Main < CC::Engine::Analyzers::Base
 
           def process_file(file)
             RubyParser.new.process(File.binread(file), file, TIMEOUT)
+          rescue Timeout::Error
```

Comment:
Ah! True... any reason why `CommandLineRunner` shouldn't rescue the timeout and raise `CC::Engine::Analyzers::ParserError`?